kubernetes/resource_kubernetes_persistent_volume_test.go

```diff
@@ -1065,6 +1090,47 @@ resource "aws_ebs_volume" "test" {
 `, name, zone, diskName)
 }
 
+func testAccKubernetesPersistentVolumeConfig_azure_basic(name, location string) string {
+	return fmt.Sprintf(`provider "azurerm" {
+  version = ">=2.20.0"
```

Comment:
no reason, really, I just grabbed it from one of the examples. I had kept it since this config isn't compatible with older versions of the Azure provider. But I'll get rid of it since it's not necessary.